src/store/transactions/withdraw/withdraw.thunks.ts

```diff
@@ -202,22 +190,27 @@ function handleTransactionSuccess(dispatch: AppDispatch, accountIndex: string) {
 
 function handleTransactionFailure(dispatch: AppDispatch, error: unknown) {
   dispatch(withdrawActions.stopTransactionApproval());
-  const withdrawAlreadyDoneErrorCode = "WITHDRAW_ALREADY_DONE";
-  const errorMsg = adapters.parseError(error);
-  dispatch(
-    openSnackbar(
-      errorMsg.includes(withdrawAlreadyDoneErrorCode)
-        ? {
-            type: "info-msg",
-            text: "The withdraw has already been done",
-          }
-        : {
-            type: "error",
-            raw: error,
-            parsed: errorMsg,
-          }
-    )
-  );
+  if (adapters.isMetamaskUserRejectedRequestError(error) === false) {
+    const withdrawAlreadyDoneErrorCode = "WITHDRAW_ALREADY_DONE";
+    adapters
+      .parseError(error)
+      .then((errorMsg) => {
+        dispatch(
+          openSnackbar(
```

Comment:
Same as before, why we don't call the `processError` here?

Reply:
Same answer applies, the message `"The withdraw has already been done"` is not reportable. It's only an "info-message" we show to the user.